### ISSUE TITLE
Random characters disappearing using sanitize action in Adobe Acrobat PRO DC - on a pdf generated with wkhtmltopdf

### ISSUE THREAD

**Issue by arnigu** [wkhtmltopdf/wkhtmltopdf#4950]
**wkhtmltopdf version(s) affected**: 0.12.6 

**OS information**
Microsoft Windows 10 Enterprise
10.0.19041 Build 19041

**Description**
When using `Adobe Acrobat Pro DC v 2021.001.20135` to sanitize a PDF created using `wkhtmltopdf` command, some characters (commonly l, i and m) disappear after saving the PDF after sanitation.  The problem seems to worsen with smaller font sizes.  The problem was also reproduced in `Adobe Acrobat XI PRO` (discontinued version)

**Attached files**
[sample.pdf](https://github.com/wkhtmltopdf/wkhtmltopdf/files/5952337/sample.pdf)
[sample-sanitized.pdf](https://github.com/wkhtmltopdf/wkhtmltopdf/files/5952339/sample-sanitized.pdf)
[sample.html.txt](https://github.com/wkhtmltopdf/wkhtmltopdf/files/5952344/sample.html.txt)

showing missing characters after sanitation

**How to reproduce**

1. Convert the file `sample.html` to a pdf using command **wkhtmltopdf.exe .\sample.html .\sample.pdf**
2. Open the 'sample.pdf' file using Adobe Acrobat Pro DC
3. Perform sanitize action [See](https://helpx.adobe.com/acrobat/using/removing-sensitive-content-pdfs.html#SanitizedocumentAcrobatPro)
4. Save and close the sanitized pdf - and ReOpen

**RESULT**: Some characters are missing

**Expected behavior**
Characters should survived the sanitazion process

**Possible Solution**  
The problem seems to have something to do with smaller font sizes

**Comment by PhilterPaper** [comment]
Have you done a search of this wkHTMLtoPDF issue list? There's something that sounds a bit familiar about this that I saw years ago, although I'm not sure the term was "sanitize". I do recall something about skinny letters vanishing like that. Anyway, you might do some looking to see if the problem was already solved.

**Comment by arnigu** [comment]
Yes, i did a little search - but nothing immediately similar popped up.  Often its a matter of searching for the right terms.  I'll look it up again! Thanks.

**Comment by arnigu** [comment]
Investigating the options again, it seems that the flag `--disable-smart-shrinking       Disable the intelligent shrinking strategy` resolves the issue.

**Comment by PhilterPaper** [comment]
Yeah, that "intelligent shrinking" function has led to all sorts of problems over the years. If you have a mystery with wkHTMLtoPDF, the first thing to do is try disabling smart shrinking. I'm not sure why it would make narrow characters disappear, but it's worth trying.
